README.md

```diff
@@ -20,14 +20,34 @@ npm install react-transition-group --save
 yarn add react-transition-group
 ```
 
+## CDN / External
+
+To exclude the react-transition-group package from your primary build and use an external CDN when bundling, link
+to the following CDN:
+
+[https://unpkg.com/react-transition-group/dist/react-transition-group.min.js](https://unpkg.com/react-transition-group/dist/react-transition-group.min.js)
+
```

Comment:
I appreciate the extra documentation, however in this case I don't to give the external consumption version of the library such a prominent place in the docs, as it's not really recommended. In other others most folks shouldn't use the CDN version, but instead include this in a vendor bundle (maybe put on a cdn) via webpack. 

I'd just make a quick note that for folks that do need a cdn version they can use unpkg.com